syntax = "proto3";
package a22;



service Registry{
  
   rpc GetServerList(Message) returns (Message) {}
   rpc RegisterServer(Message) returns (Message) {}

}

message Message{
   string message = 1;
}